test/togeojson-gpx.test.js

```diff
@@ -72,10 +72,11 @@ test('[GPX togeojson] convert and index valid GPX', function(assert) {
       assert.ok(fs.existsSync(path.join(outdir, 'tracks.geojson')), 'converted layer');
       assert.ok(fs.existsSync(path.join(outdir, 'tracks.geojson.index')), 'created index');
       assert.ok(fs.existsSync(path.join(outdir, 'metadata.json')), 'added metadata of original gpx');
+      assert.ok(fs.existsSync(path.join(outdir, 'archived.gpx')), 'original file archived');
+      assert.equal(fs.readFile(infile) === fs.readFile(path.join(outdir, 'archived.gpx')), true, 'file contents are the same');
```

Comment:
See comment above about reading files synchronously vs. asynchronously. Once you make the appropriate changes, you'll notice reading the file will return a Buffer. You can convert this buffer into a string and check for equality.